subscription/upgrade.md

```diff
@@ -33,33 +33,3 @@ To purchase a Pro, Team, or Business subscription:
     > To upgrade to a Business subscription via purchase orders you must have at least 25 seats. To create a purchase order,  specify the number of users and then select the **Contact Sales form**. Then, continue the on-screen instructions for the billing details and select **Purchase**.
 
 4. Select **Continue to Payment** and update the fields to complete your purchase.
-
-## Switch from a monthly subscription to an annual subscription
```

Comment:
I see this content was removed and not migrated to the Billing section. Was it deemed unnecessary?

Reply:
@yavorg, we don't provide support switching from annually to monthly. But it would be nice to support it for our customers.

Reply:
@yavorg Content was moved to the cycle.md page :)